requirements.txt

```diff
@@ -8,4 +8,5 @@ cx_Freeze >= 6.6
 ctREFPROP
 pandas
 openpyxl
-h5py
\ No newline at end of file
+h5py
+gmsh-sdk
```

Comment:
This package is deprecated according to its PyPI [homepage](https://pypi.org/project/gmsh-sdk/).
Please use "gmsh" instead.

Reply:
I didn't see that.  the current open-pulse uses 4.6.0 gmsh.  have you done any testing on 4.11.1?